run_test.sh

```diff
@@ -15,7 +15,7 @@ ETCD_PID=$!
 echo "mode: count" > profile.cov
 
 # Standard go tooling behavior is to ignore dirs with leading underscors
-for dir in $(find . -maxdepth 10 -not -path './.git*' -not -path '*/_*' -not -path './vendor/*' -not -path '*/test_data/*' -not -path '*/goext_example*' -type d);
+for dir in $(find . -maxdepth 10 -not -path './.git*' -not -path '*/_*' -not -path './vendor/*' -not -path '*/test_data/*' -not -path '*/goext_example*' -type d | sort);
```

Comment:
This is also the second reason why the unit tests are randomly picked. By using `sort` command, the order of running tests become the same everytime.

Reply:
Are we depending somewhere on the order of test execution?